examples/imageviewer/shared/src/iosMain/kotlin/example/imageviewer/ImageViewer.ios.kt

```diff
@@ -57,7 +57,10 @@ fun getDependencies(ioScope: CoroutineScope, toastState: MutableState<ToastState
                         val window = UIApplication.sharedApplication.windows.last() as? UIWindow
                         val currentViewController = window?.rootViewController
                         val activityViewController = UIActivityViewController(
-                            activityItems = listOf(UIImage(data = data)),
+                            activityItems = listOf(
+                                UIImage(data = data),
+                                picture.name + "\n"  + picture.description
```

Comment:
I think we don't need picture.name for sharing. What about Android?

Reply:
Done